pl_bolts/datamodules/async_dataloader.py

```diff
@@ -80,16 +88,19 @@ def load_instance(self, sample):
         else:
             return sample
 
-    def __iter__(self):
+    def __iter__(self) -> "AsynchronousLoader":
         # We don't want to run the thread more than once
         # Start a new thread if we are at the beginning of a new epoch, and our current worker is dead
-        if (not hasattr(self, 'worker') or not self.worker.is_alive()) and self.queue.empty() and self.idx == 0:
+
+        # yapf: disable
+        if (not hasattr(self, 'worker') or not self.worker.is_alive()) and self.queue.empty() and self.idx == 0:  # type: ignore[has-type] # noqa: E501
             self.worker = Thread(target=self.load_loop)
+            # yapf: enable
```

Comment:
Just for the record, I had a collision here between yapf and flake8. https://github.com/PyTorchLightning/pytorch-lightning/pull/5591

Reply:
Applying `yapf` will cause flake8's error `[W503] line break before binary operator`.